workflow/controller/operator_test.go

```diff
@@ -3379,6 +3379,395 @@ func TestRetryNodeOutputs(t *testing.T) {
 	assert.Contains(t, scope.scope, "steps.influx.finishedAt")
 }
 
+var workflowWithPVCAndFailingStep = `
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  creationTimestamp: "2020-10-14T18:12:34Z"
+  generation: 17
+  labels:
+    workflows.argoproj.io/completed: "true"
+    workflows.argoproj.io/phase: Failed
+  name: wf-with-pvc
+  resourceVersion: "8196"
+  selfLink: /apis/argoproj.io/v1alpha1/namespaces/argo-next/workflows/wf-with-pvc
+  uid: 01996a37-718c-48b6-9038-33016646d425
+spec:
+  arguments: {}
+  entrypoint: entrypoint
+  templates:
+  - arguments: {}
+    inputs: {}
+    metadata: {}
+    name: entrypoint
+    outputs: {}
+    retryStrategy:
+      limit: 1
+    steps:
+    - - arguments: {}
```

Comment:
any wayto make this easier to understand? it's just see lots and lots of very long YAML  and I don't know what is trying to be tested:

* reduce YAML to the minimum that is need to test bug